service/k8sclient/kubectl/kubectl.go

```diff
@@ -0,0 +1,61 @@
+// dbaas-controller
+// Copyright (C) 2020 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+package kubectl
+
+import (
+	"bytes"
+	"context"
+	"io"
+	"os/exec"
+	"time"
+
+	"github.com/percona-platform/dbaas-controller/logger"
+)
+
+var cmd = []string{"minikube", "kubectl", "--"} //nolint:gochecknoglobals
```

Comment:
https://jira.percona.com/browse/PMM-6212 should be done soon, so let's add a basic `kubectl` binary selection code

Reply:
Working on it